Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1481 @@
+    close_source_branch = args.get('close_source_branch', None)
+    body = create_pull_request_body(title, source_branch, destination_branch, reviewer_id, description,
+                                    close_source_branch)
+    if not repo:
+        repo = client.repository
+    response = client.pull_request_create_request(repo, body)
+    return CommandResults(readable_output='The pull request was created successfully',
+                          outputs_prefix='Bitbucket.PullRequest',
+                          outputs=response,
+                          raw_response=response)
+
+
+def pull_request_update_command(client: Client, args: Dict) -> CommandResults:
+    """ Updates issues from Bitbucket. If a certain argument isn't given, don't update it on the issue
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a dictionary contains information about the updated pull request.
+    """
+    repo = args.get('repo')
+    pull_request_id = args.get('pull_request_id', '')
+    title = args.get('title', None)
+    source_branch = args.get('source_branch', None)
+    destination_branch = args.get('destination_branch', None)
+    reviewer_id = args.get('reviewer_id', None)
+    description = args.get('description', None)
+    close_source_branch = args.get('close_source_branch', None)
+    body = create_pull_request_body(title, source_branch, destination_branch, reviewer_id, description,
+                                    close_source_branch)
+    if not repo:
+        repo = client.repository
+    response = client.pull_request_update_request(repo, body, pull_request_id)
+    return CommandResults(readable_output=f'The pull request {pull_request_id} was updated successfully',
+                          outputs_prefix='Bitbucket.PullRequest',
+                          outputs=response,
+                          raw_response=response)
+
+
+def pull_request_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of pull requests.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a dictionary contains a list of pull request.
+        If a state is provided than the list will contain only PR with the wanted status.
+        If a state is not provided, by default a list of the open pull requests will return.
+    """
+    repo = args.get('repo')
+    pull_request_id = args.get('pull_request_id', None)
+    state = args.get('state', None)
+    limit = int(args.get('limit', 50))
+    page = int(args.get('page', 1))
+    check_args(limit, page)
+    page_size = min(100, limit)
+    params = {
+        'page': page,
+        'pagelen': page_size
+    }
+    if state:
+        params["state"] = state
+    if not repo:
+        repo = client.repository
+    response = client.pull_request_list_request(repo, pull_request_id, params)
```

Comment:
```suggestion
    response = client.pull_request_list_request(repo, pull_request_id, page, page_size, state)
```
In the request use assign_params
It is easier to debug when we see the parameters that we pass